libr/core/cmd_type.c

```diff
@@ -470,40 +471,41 @@ static int cmd_type(void *data, const char *input) {
 			break;
 		}
 		break;
-	case 's': // "ts"
+	case 's': { // "ts"
+		char *name = strchr (input + 2, ' ');
 		switch (input[1]) {
 		case '?':
 			r_core_cmd_help (core, help_msg_ts);
 			break;
 		case '*':
-			showFormat (core, input + 2, 1);
+			showFormat (core, name + 1, 1);
 			break;
 		case ' ':
 			showFormat (core, input + 2, 0);
```

Comment:
Maybe change here to `name` too?

Reply:
No since that case is a space already , name would be null there !

Reply:
Yup done !